### ISSUE TITLE
Documentation on replacing navigation component for version 3.1 seems to be incorrect

### ISSUE THREAD

**Issue by alexandis** [abpframework/abp#5642]
3.1.2 on Angular.

We use custom Navigation component and successfully replaced it for version 2.9.0 according to your manual.
When we have upgraded to 3.1.2 and used the corresponding manual: https://docs.abp.io/en/abp/latest/UI/Angular/Component-Replacement - it appeared the markup is now broken and Navigation area is not displayed at all. We can't figure out, what exactly is wrong. You seems to have changed the markup hierarchy, probably it's a bug in description, probably it's inconsistent CSS now...

**Comment by alexandis** [comment]
Small update: when I have added the following code from old markup to wrap existing `<ul class="navbar-nav">...</ul>`:

```
<abp-nav-items ng-reflect-small-screen="false" class="ng-star-inserted">
  <nav class="navbar navbar-expand-lg user-nav-mobile">
    <div id="navbarToolbar" class="navbar-collapse d-lg-block toolbar-nav-wrapper collapse">
    ...
    </div>
  </nav>
</abp-nav-items>
```

the markup displayed more or less properly. But then suddently I have noticed, that no matter what I add or remove to this markup - it is ignored and standard components (Language, User profile) is displayed. My markup is ignored.

I believe it this - as well as other markup issues now - have something to do with Lepton theme. We use this (not basic theme as represented in guide):

`import { eThemeLeptonComponents } from '@volo/abp.ng.theme.lepton/lib/enums/components';`

Could you please clarify what needs to be changed to make markup work properly in whole Angular site?

**Comment by bnymncoskuner** [comment]
Hello,

We've improved the template of `abp-nav-items` in lepton theme. It will be available with version 3.3.0-rc.1. When you upgrade your packages, you should be able to use any template you want and it'll be placed at the top-right corner of the screen. Also, using `<abp-nav-items>....</abp-nav-items>` will not work because that component does not render any content passed from outside. That's why your template is being ignored and default items are displayed.
